src/kagglehub/datasets_helpers.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from http import HTTPStatus
+from typing import List
+
+from kagglehub.clients import KaggleApiV1Client
+from kagglehub.exceptions import KaggleApiHTTPError
+from kagglehub.handle import DatasetHandle
+
+logger = logging.getLogger(__name__)
+
+
+def _create_dataset(owner_slug: str, dataset_slug: str) -> None:
+    data = {"ownerSlug": owner_slug, "slug": dataset_slug, "title": dataset_slug, "isPrivate": True}
+    api_client = KaggleApiV1Client()
+    api_client.post("/datasets/create/new", data)
+    logger.info(f"Dataset '{dataset_slug}' Created.")
+
+
+def _create_dataset_instance(dataset_handle: DatasetHandle, files: List[str]) -> None:
+    data = {"files": [{"token": file_token} for file_token in files]}
+    api_client = KaggleApiV1Client()
+    api_client.post("/datasets/create/new", data)
+    logger.info(
+        f"Your dataset instance has been created.\nFiles are being processed...\nSee at: {dataset_handle.to_url()}"
+    )
+
+
+def _create_dataset_instance_version(dataset_handle: DatasetHandle, files: List[str], version_notes: str = "") -> None:
+    data = {"versionNotes": version_notes, "files": [{"token": file_token} for file_token in files]}
+    api_client = KaggleApiV1Client()
+    api_client.post(f"/datasets/{dataset_handle.owner}/{dataset_handle.dataset_name}/create/", data)
```

Comment:
I don't see where this URL is supported, only `/datasets/create/version/owner/slug`.  Where is this one registered?

Reply:
I used the format of models as a placeholder and forgot to update this url